classes/plugins/importexport/PKPImportExportDeployment.inc.php

```diff
@@ -213,8 +213,13 @@ function setFileDBIds($fileDBIds) {
 	 * @return integer
 	 */
 	function getFileDBId($fileId, $revisionId) {
```

Comment:
I think they are -- earlier, the result was returned only if there was an entry for a matching fileId and revisionId. Now, also a case where only matching fileId is found is considered -- that case is when there are more than one revision for a file to be imported -- earlier each revision was inserted as a new file i.e. with a new fileId.